### ISSUE TITLE
Expose HTTP Writer As Available Writer

### ISSUE THREAD

**Issue by inrix-adasgupta** [jmxtrans/jmxtrans#583]
Please expose a basic HTTP writer. It should be configurable for host, port, method and optionally for security. A templating option should also be provided. As a start a basic JSON, or CSV format can be provided with package before supporting templating.

It'll be very useful for many currently non-supporting tools, e.g. SumoLogic. Also custom app can easily be built to consume metrics.

**Comment by gehel** [comment]
There is already an [HttpOutputWriter](https://github.com/jmxtrans/jmxtrans/blob/master/jmxtrans-output/jmxtrans-output-core/src/main/java/com/googlecode/jmxtrans/model/output/support/HttpOutputWriter.java). It is not directly configureable, but it can be reused to write a simple Java class for whatever format you need. See for example [LibratoWriter2](https://github.com/jmxtrans/jmxtrans/blob/master/jmxtrans-output/jmxtrans-output-core/src/main/java/com/googlecode/jmxtrans/model/output/LibratoWriter2.java). Yes, this requires writing code and not just configuration (but I tend to think that configuration is just code that you cannot easily unit test).

If you are interested in providing a more flexible approach, feel free to submit a pull request!

**Comment by ls0f** [comment]
@gehel has  plan to implement this?

**Comment by gehel** [comment]
No, I have no plan on implementing this. I usually don't write new OutputWriters for which I don't have a use case personally, as I would probably get the constraints wrong (and I don't have all that much free time). That being said, I'm happy to accept pull requests!
